packages/compass-connections-navigation/src/connections-navigation-tree.tsx

```diff
@@ -454,7 +487,7 @@ const ConnectionsNavigationTree: React.FunctionComponent<
   const [rootProps, currentTabbable] = useVirtualNavigationTree<HTMLDivElement>(
     {
       items: items as NavigationTreeData,
-      activeItemId: activeNamespace,
+      activeItemId: activeNamespace || '', // TODO
```

Comment:
What will this todo be?

Reply:
See the open questions in description ;) tldr; I didn't include this bit in the original ticket scope and only stumbled on it during implementation

Reply:
added ticket number in the comment

Reply:
Ahh got it, thank you 😄